### ISSUE TITLE
Feature request: Deselect all

### ISSUE THREAD

**Issue by JonathanSchubert** [sn-extensions/simple-task-editor#21]
It would be great to have a button "Deselect All". Pushing this button would move all completed tasks back to the open tasks list.

Motivation:
I often use this extension for packing lists, so that I don't forget anything when packing for vacation/gym. If the list is quite long, it is quite annoying to uncheck all entries.

**Comment by mobitar** [comment]
Yeah, actually got this feedback before for similar use cases. Hoping to build on this editor in the future to allow for things like this.
